core/src/main/scala/scala/pickling/Pickler.scala

```diff
@@ -83,15 +84,18 @@ trait Unpickler[T] {
   /** The fast type tag associated with this unpickler. */
   def tag: FastTypeTag[T]
 }
+/** Shim for Java code. */
```

Comment:
Doesn't seem useful as a Scaladoc comment, should this instead be an implementation comment?